docs/tutorial/4-template.md

```diff
@@ -240,30 +236,40 @@ The resulting template function looks like this:
   doc,
 ) = {
   // Set and show rules from before.
->>> set page(columns: 2)
+>>> // (skipped)
 <<<   ...
 
-  set align(center)
-  text(17pt, title)
-
-  let count = authors.len()
-  let ncols = calc.min(count, 3)
-  grid(
-    columns: (1fr,) * ncols,
-    row-gutter: 24pt,
-    ..authors.map(author => [
-      #author.name \
-      #author.affiliation \
-      #link("mailto:" + author.email)
-    ]),
-  )
-
-  par(justify: false)[
-    *Abstract* \
-    #abstract
+>>> place(
+>>>   top + center,
+>>>   float: true,
+>>>   scope: "parent",
+>>>   clearance: 2em,
+>>> )[
+<<<   place(...)[
```

Comment:
Not sure whether it's worth to hide these. Currently, it looks a little odd and doesn't really help clarity much I think:
```
  place(
    ...,
    {
```